### PR TITLE
Add puma dashboard

### PR DESCRIPTION
**Pull request by matsimitsu**
Add puma dashboard

### CODE REVIEW

File: dashboards/puma/main.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - title: Pool capacity
+      line_label: '%hostname%'
+      metrics:
+        - name: puma_pool_capacity
+          fields:
+            - GAUGE
+          tags:
+            hostname: '*'
+    - title: Puma worker info
+      line_label: '%hostname% - %kind%'
```
Comment (jvanbaarsen) [inline dashboards/puma/main.yml]: I think this should also be `type` right?

File: dashboards/puma/main.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  - puma_threads
+dashboard:
+  title: Puma metrics
+  description: |
+    Track Puma workers on this dashboard, 
+    see how many worker processes are running 
+    and what the used/available concurrency is for those workers. 
+  graphs:
+    - title: Threads running/max/idle
+      line_label: '%hostname% - %type%'
```
Comment (tombruijn) [inline dashboards/puma/main.yml]: lengthy hostnames can make this less readable. In my demo dashboard I added the hostname last so the important bit (the type) is first.

```suggestion
      line_label: '%type% - %hostname%'
```

File: dashboards/puma/main.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    - title: Pool capacity
+      line_label: '%hostname%'
+      metrics:
+        - name: puma_pool_capacity
+          fields:
+            - GAUGE
+          tags:
+            hostname: '*'
+    - title: Puma worker info
+      line_label: '%hostname% - %kind%'
```
Comment (matsimitsu) [inline dashboards/puma/main.yml]:
```suggestion
      line_label: '%hostname% - %type%'
```

File: dashboards/puma/main.yml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    - title: Pool capacity
+      line_label: '%hostname%'
+      metrics:
+        - name: puma_pool_capacity
+          fields:
+            - GAUGE
+          tags:
+            hostname: '*'
+    - title: Puma worker info
+      line_label: '%hostname% - %type%'
```
Comment (tombruijn) [inline dashboards/puma/main.yml]: this one still begins with hostname.

**Pull request merged**